packages/@aws-cdk/core/lib/stack-synthesizers/default-synthesizer.ts

```diff
@@ -91,6 +97,20 @@ export interface DefaultStackSynthesizerProps {
    */
   readonly lookupRoleArn?: string;
 
+  /**
+   * External ID to use when assuming lookup role
+   *
+   * @default - No external ID
+   */
+  readonly lookupRoleExternalId?: string;
+
+  /**
+   * Whether or not to disable use of the lookup role
+   *
+   * @default - false
+   */
+  readonly disableLookupRole?: boolean;
```

Comment:
In general, I hate to use flags that are named after a negative. In frail human heads it becomes very hard to reason about (at least in mine 😅).

Can we turn this into something like this (adding some explanation of what the flag does as well):

```suggestion
  /**
   * Use the bootstrapped lookup role for (read-only) stack operations
   *
   * Use the lookup role when performing a `cdk diff`. If set to `false`, the
   * `deploy role` credentials will be used to perform a `cdk diff.`
   *
   * Requires bootstrap stack version 8.
   *
   * @default true
   */
  readonly useLookupRoleForStackOperations?: boolean;
```

Reply:
Updated.